Library/Homebrew/utils/ast.rb

```diff
@@ -190,6 +190,19 @@ def add_stanza(name, value, type: nil)
         tree_rewriter.insert_after(preceding_expr, "\n#{stanza_text(name, value, indent: 2)}")
       end
 
+      sig { returns(T::Boolean) }
+      def include_runtime_cpu_detection?
+        install_node = children.find do |child|
+          (child.is_a? RuboCop::AST::DefNode) && child.method_name == :install
```

Comment:
```suggestion
          child.def_type? && child.method_name == :install
```

Probably better than a `is_a?` check

Reply:
@Bo98 
I agree with you. However, this change would lead to a `brew typecheck` error.

```
brew typecheck
utils/ast.rb:196: Method method_name does not exist on RuboCop::AST::Node https://srb.help/7003
     196 |          child.def_type? && child.method_name == :install
                                             ^^^^^^^^^^^
  Got RuboCop::AST::Node originating from:
    utils/ast.rb:195:
     195 |        install_node = children.find do |child|
                                                   ^^^^^
  Did you mean method_node? Use `-a` to autocorrect
    utils/ast.rb:196: Replace with method_node
     196 |          child.def_type? && child.method_name == :install
                                             ^^^^^^^^^^^
    sorbet/rbi/hidden-definitions/hidden.rbi:4934: Defined here
    4934 |  def method_node(param0=T.unsafe(nil)); end
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Errors: 1
Check https://docs.brew.sh/Typechecking for more information on how to resolve these errors.
```

Reply:
Ah ok. Makes sense that Tapioca doesn't recognise `*_type?`. Ignore this then.